### ISSUE TITLE
Upload stable release

### ISSUE THREAD

**Issue by ivan-ristovic** [matf-pp/2021_Psi-Ascii-#5]
A stable binary should also be uploaded: https://github.com/matf-pp/2021_Psi-Ascii-/releases/new

**Comment by idzii** [comment]
What is stable binary? Is it just compiled file that can be executed or something else?

**Comment by ivan-ristovic** [comment]
Yes. It would be easier for someone to download and run that executable instead of recompiling the source, especially if it requires extra dependencies...

**Comment by ivan-ristovic** [comment]
I accidentally closed this issue, reopening...
